src/Symfony/Component/Scheduler/RecurringMessage.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Scheduler;
+
+use Symfony\Component\Scheduler\Exception\LogicException;
+use Symfony\Component\Scheduler\Trigger\CallbackTrigger;
+use Symfony\Component\Scheduler\Trigger\CronExpressionTrigger;
+use Symfony\Component\Scheduler\Trigger\PeriodicalTrigger;
+use Symfony\Component\Scheduler\Trigger\TriggerInterface;
+
+/**
+ * @experimental
+ */
+final class RecurringMessage
```

Comment:
the class should not be final to me, because it's part of the signature or other classes and doing so kills some extensibiiity

Reply:
done

Reply:
More restrictions are good to start. When we will see a reason to remove "filnal", it's easy to remove it. But it's not easy to [add](https://github.com/symfony/symfony/issues/47096#issuecomment-1434333688) it.

I dont's see any usable way to exnend it, so I think it should be final.

Reply:
It's final again :)